spec/support/base_model.cr

```diff
@@ -0,0 +1,5 @@
+class BaseModel < Avram::Model
+  def self.database
+    TestDatabase
+  end
```

Comment:
Here is how you set what database a model uses. Can be done on the abstract class or could be overridden per model